QUARKUS-6965.md

```diff
@@ -0,0 +1,41 @@
+# QUARKUS-6965 Keep the Hibernate ORM and Reactive extensions enabled even when no entity is defined
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6965
+
+Upstream PRs:
+* https://github.com/quarkusio/quarkus/pull/48732 - main PR for the new feature
+* https://github.com/quarkusio/quarkus/pull/48783
+* https://github.com/quarkusio/quarkus/pull/50262
+
+New feature allow to start Hibernate ORM/reactive even if no entity is defined to use it.
+It is intended for users that want to use Hibernate for direct usage of database through Hibernate,
+for cases like executing native queries using `StatelessSession`.
+It still requires a PU, and it's datasource to be active, otherwise Hibernate will not start.
+
+New feature is primarily implemented in the PR [48732](https://github.com/quarkusio/quarkus/pull/48732).
+Other PRs [48783](https://github.com/quarkusio/quarkus/pull/48783) and [50262](https://github.com/quarkusio/quarkus/pull/50262) are mostly about behaviour tweaks and fixes. 
+
+## Scope of testing
+Verify the feature to have Hibernate ORM/reactive started without any entity defined.
+This will require creating test app, which will use no Hibernate entities, but will allow native SQL queries.
+
+Starting Hibernate can be disabled at build time by setting `quarkus.hibernate-orm.enabled=false`.
+Also, it will also be disabled at runtime if no datasource is active.
+
+Goal is to verify working scenario with entity-less app as well as scenarios where Hibernate is not started.
```

Comment:
I wonder if the most interesting use case wouldn't be a Hibernate repository without any entity. Was it possible before? I guess it wasn't since the Hibernate wasn't active. See https://docs.hibernate.org/stable/orm/repositories/html_single/#query_method you can do something like (my metacode):

```java
@Repository
public interface MyRepository {

  record FruitView(Long id, String name) {
    }

 // native query here   
 @SQL("SELECT f.id, f.name FROM Fruit f")
    List<FruitView> listFruits();

}
```